source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss

```diff
@@ -187,18 +207,30 @@
 .table {
   @extend %regularText;
   margin-bottom: 14px;
+  display: flex;
```

Comment:
@daniloprates please sort these CSS rules alphabetically 🙏

Reply:
@daniloprates fixed in: https://github.com/input-output-hk/daedalus/pull/1420/commits/425f483ad1bfb715ab18770650d090921d7ef431